test/types/schema.test.ts

```diff
@@ -388,9 +388,9 @@ export function autoTypedSchema() {
     array3?: any[];
     array4?: any[];
     array5?: any[];
-    decimal1?: Schema.Types.Decimal128;
-    decimal2?: Schema.Types.Decimal128;
-    decimal3?: Schema.Types.Decimal128;
+    decimal1?: Types.Decimal128;
```

Comment:
This is correct according to the docs. "Types.Decimal128" instead of "Schema.Types.Decimal128"